src/test/java/net/minecraftforge/debug/DynBucketTest.java

```diff
@@ -68,24 +68,32 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+import static net.minecraftforge.debug.DynBucketTest.ObjectHolders.*;
+
 @Mod(modid = DynBucketTest.MODID, name = "DynBucketTest", version = "0.1", dependencies = "after:" + ModelFluidDebug.MODID, acceptableRemoteVersions = "*")
 public class DynBucketTest
 {
     public static final String MODID = "dynbuckettest";
+
     private static final ResourceLocation simpleTankName = new ResourceLocation(MODID, "simpletank");
     private static final ResourceLocation testItemName = new ResourceLocation(MODID, "testitem");
 
-    private static final boolean ENABLE = true;
+    private static final boolean ENABLE = false;
+
     private static Logger logger;
 
-    @ObjectHolder("testitem")
-    public static final Item TEST_ITEM = null;
-    @ObjectHolder("simpletank")
-    public static final Block TANK_BLOCK = null;
-    @ObjectHolder("simpletank")
-    public static final Item TANK_ITEM = null;
-    @ObjectHolder("dynbottle")
-    public static final Item DYN_BOTTLE = null;
+    @ObjectHolder(MODID)
+    public static final class ObjectHolders
+    {
+        @ObjectHolder("testitem")
+        public static final Item TEST_ITEM = null;
```

Comment:
Ok, my bad. I was going off what the docs said, but apparently the docs are wrong about this.